cli/ballerina-packerina/src/test/resources/test-resources/maven-dependency-project/project-without-custom-maven-dependencies/Ballerina.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+org-name= "suganya"
+version= "0.1.0"
+
+[dependencies]
+
+[platform]
+target = "java8"
+
+    [[platform.repositories]]
```

Comment:
For this scenario, Can't we generate warning message including the missing fields and continue the process? Then the user can continue works or if he or she misses it without knowing can fill it.

Reply:
@lnash94, Thanks for pointing out. We need to throw an exception if the repository defined with the empty body.